deploy/vagrant/Vagrantfile

```diff
@@ -27,15 +28,19 @@ Vagrant.require_version ">= 1.6.5"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   (1..Total).each do |i|
     # multi vm config
-    if i == Total
-      name = "TachyonMaster"
+    if i <= TachyonV.masters
+      if i == 1
+        name = "TachyonMaster"
```

Comment:
I think the names should be TachyonMaster1, TachyonMaster2, and so on if # masters is > 1.